.circleci/config.yml

```diff
@@ -299,22 +299,22 @@ jobs:
       - install-prerequisites
       - restore_cache:
           keys:
-            - GraphBLAS-mac-{{ checksum "deps/GraphBLAS/README.md" }}
+            - GraphBLAS-mac-clang-{{ checksum "deps/GraphBLAS/README.md" }}
       - run:
           name: Build
           shell: /bin/bash -l -eo pipefail
           command: |
-            make all package
+            DYLD_LIBRARY_PATH=/usr/local/opt/libomp/lib CC=clang make all package
       - save_cache:
           paths:
             - deps/GraphBLAS/build
-          key: GraphBLAS-mac-{{ checksum "deps/GraphBLAS/README.md" }}
+          key: GraphBLAS-mac-clang-{{ checksum "deps/GraphBLAS/README.md" }}
       - run:
           name: Test
           shell: /bin/bash -l -eo pipefail
           command: |
             ulimit -n 10000
-            make test
+            DYLD_LIBRARY_PATH=/usr/local/opt/libomp/lib CC=clang CXX=clang++ make test
```

Comment:
Can we use the direct /usr/local/lib here? Brew should be installing its taps there.